features/step_definitions/basic_steps.rb

```diff
@@ -34,6 +34,10 @@
   end
 end
 
+Given(/^I am on the quizmaster Add questions page for "([^"]*)"$/) do |quiz|
+  quiz = Quiz.find_by(name: name)
+  visit quizmaster_add_questions_path(quiz)
+end
```

Comment:
This test goes to a route that doesn't exist, right? What is this add_questions path? Is this step not being used? If so, lose it.